### PR TITLE
Integration of video conversion services in EndPoint

### PR DESCRIPTION
**Pull request by Milena-Rodriguez-Montecinos**

### CODE REVIEW

File: CONVERTER_SERVICE/.env [hunk 1]
```diff
@@ -1,5 +1,7 @@
...
-ZIP_PATH=C:/Users/serqi/Desktop/Fundacion/proga_101/project/AT14-SERVICES/CONVERTER_SERVICE/resources/zip/
-BASE_URL="http://localhost:8080/files/"
-VIDEO_PATH = C:/Users/serqi/Desktop/Fundacion/proga_101/project/AT14-SERVICES/CONVERTER_SERVICE/resources/upload/
+CONVERTER_PATH=C:/Users/mile_/OneDrive/Documentos/Develop/AT14-SERVICES/CONVERTER_SERVICE/thirdParty/ffmpeg.exe
+OUTPUT_PATH=C:/Users/mile_/OneDrive/Documentos/Develop/AT14-SERVICES/CONVERTER_SERVICE/resources/outputPath/
+UPLOAD_PATH=C:/Users/mile_/OneDrive/Documentos/Develop/AT14-SERVICES/CONVERTER_SERVICE/resources/upload/
+VIDEO_PATH=C:/Users/mile_/OneDrive/Documentos/Develop/AT14-SERVICES/CONVERTER_SERVICE/resources/video/
+AUDIO_PATH=C:/Users/mile_/OneDrive/Documentos/Develop/AT14-SERVICES/CONVERTER_SERVICE/resources/audio/
+ZIP_PATH=C:/Users/mile_/OneDrive/Documentos/Develop/AT14-SERVICES/CONVERTER_SERVICE/resources/zip/
+BASE_URL="http://localhost:8080/files/"
```
Comment (serqiiiio) [inline CONVERTER_SERVICE/.env:7]: The last line space required

File: CONVERTER_SERVICE/src/controller/videoController.js [hunk 2]
```diff
@@ -63,47 +21,8 @@ const changeVideoFormat = async (req, res) => {
...
-        name: req.file.originalname,
-        path: resultPathVideoFormat,
-        checksum: hash,
-    });
-    fileModel.save(function (err, doc) {
-        if (err) return console.error(err);
-        console.log("Document inserted successfully!");
-    });
-
-    //zipping.zipDownload(req, res);
```
Comment (serqiiiio) [inline CONVERTER_SERVICE/src/controller/videoController.js:77]: Have to erase comment

File: CONVERTER_SERVICE/src/middleware/videoService.js [hunk 3]
```diff
@@ -76,4 +77,4 @@ class VideoServices {
     }  
 }
 
-module.exports = VideoServices;
\ No newline at end of file
+module.exports = VideoServices;
```
Comment (serqiiiio) [inline CONVERTER_SERVICE/src/middleware/videoService.js:80]: Add the last line required

File: CONVERTER_SERVICE/src/routes/index.js [hunk 4]
```diff
@@ -1,14 +1,14 @@
 const express = require("express");
 const router = express.Router();
-const controllerImagenes = require("../controller/ImagesController");
+//const controllerImagenes = require("../controller/ImagesController");
```
Comment (serqiiiio) [inline CONVERTER_SERVICE/src/routes/index.js:3]: Erase the comments

File: CONVERTER_SERVICE/src/routes/index.js [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
 const changeVideoFormat = require("../controller/VideoController");
-const controller = require("../controller/file.controller");
+//const controller = require("../controller/file.controller");
```
Comment (serqiiiio) [inline CONVERTER_SERVICE/src/routes/index.js:5]: Erase the comments

**Review by serqiiiio — CHANGES_REQUESTED** [review]
Please erase the videos, and the zips are additional files

File: CONVERTER_SERVICE/src/controller/videoController.js (hunk 2 above)
Comment (serqiiiio) [inline CONVERTER_SERVICE/src/controller/videoController.js:77]: erase comments

**Review by serqiiiio — APPROVED** [review]

**Review by edwinPato — APPROVED** [review]

**Pull request merged**